src/content/en/tools/chrome-devtools/profile/network-performance/resource-loading.markdown

```diff
@@ -385,122 +388,213 @@ The Frames tab shows messages sent or received over a WebSocket connection. This
+  A HAR file contains a JSON data structure that describes the network 
+  "waterfall". Several [third-party](https://ericduran.github.io/chromeHAR/) 
+  [tools](https://code.google.com/p/harviewer/) can reconstruct the network 
+  waterfall from the data in the HAR file. See 
+  [Web Performance Power Tool: HTTP Archive 
+  (HAR)](https://www.igvita.com/2012/08/28/web-performance-power-tool-http-archive-har/)
+  for more information.
+* **Save as HAR with Content**: Saves all network data to an
+  HAR file along with each page resource. Binary resources, including images, 
+  are encoded as Base64-encoded text.
+* **Clear Browser Cache**: Clear the browser cache.
+* **Clear Browser Cookies**: Clear the browser's cookies.
+* **Open in Sources Panel**: Open the selected resource in the **Sources** 
+  panel.
+* **Open Link in New Tab**: Opens the selected resource in a new tab. You 
+  can also double-click the resource name in the Network table.
+* **Copy Link Address**: Copies the resource URL to the system clipboard.
+* **Save**: Save the selected text resource. Only displayed on text 
+  resources.
+* **Replay XHR**: Re-send the selected `XMLHTTPRequest`. Only displayed on XHR
+  resources.
 
 ### Copy requests as cURL commands
 
-[cURL](http://curl.haxx.se/) is a command line tool for making HTTP transactions. The Network panel's **Copy as cURL** command recreates an HTTP request (including HTTP headers, SSL certificates, and query string parameters) and copies it as a cURL command string to the clipboard. You can then paste the string into a terminal window (on a system with cURL) to execute the same request.
+[cURL](http://curl.haxx.se/) is a command line tool for making HTTP 
+transactions. When you right-click on a resource in the **Requests Table** and
+then select **Copy as cURL**, DevTools recreates an HTTP request (including
+HTTP headers, SSL certificates, and query string parameters) and copies it 
+as a cURL command string to the clipboard. You can then paste the string 
+into a terminal window (on a system with cURL) to execute the same request.
 
-Below is an example cURL command line string taken from a XHR request on the Google News home page.
+Below is an example cURL command line string taken from a XHR request on 
+the Google News home page.
 
 `curl 'http://news.google.com/news/xhrd=us' -H 'Accept-Encoding: gzip,deflate,:sdch' -H 'Host: news.google.com' -H 'Accept-Language: en-US,en;q=0.8' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_8_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/29.0.1510.0 Safari/537.36' -H 'Accept: */*' -H 'Referer: http://news.google.com/nwshp?hl=en&tab=wn' -H 'Cookie: NID=67=eruHSUtoIQA-HldQn7U7G5meGuvZOcY32ixQktdgU1qSz7StUDIjC_Knit2xEcWRa-e8CuvmADminmn6h2_IRpk9rWgWMdRj4np3-DM_ssgfeshItriiKsiEXJVfra4n; PREF=ID=a38f960566524d92:U=af866b8c07132db6:FF=0:TM=1369068317:LM=1369068321:S=vVkfXySFmOcAom1K' -H 'Connection: keep-alive' --compressed`
```

Comment:
Maybe run this through the [bash highlighter](https://web-central.appspot.com/web/resources/jekyll/useful-and-common-tags?hl=en#highlight) to see if it helps the output look better at all.